disperser/batcher/metrics.go

```diff
@@ -115,9 +121,28 @@ func NewMetrics(httpPort string, logger common.Logger) *Metrics {
 		),
 	}
 
+	finalizerMetrics := FinalizerMetrics{
+		NumBlobs: promauto.With(reg).NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: namespace,
+				Name:      "num_blobs",
```

Comment:
Name should specify it's a metric from finalizer. Maybe "finalizer_num_blobs"?

Reply:
done